src/scribe_data/cli/main.py

```diff
@@ -166,8 +166,9 @@ def main() -> None:
     get_parser.add_argument(
         "-wdp",
         "--wikidata-dump-path",
-        type=str,
-        help="Path to a local Wikidata lexemes dump for running with '--all'.",
+        nargs="?",
+        const="",
+        help="Path to a local Wikidata lexemes dump. Uses default directory if no path provided.",
```

Comment:
This here made me wonder.. 
Should we specify in the help message what the default path/directory would be? For this and other default directories as well?